docs/guides/functions.ipynb

```diff
@@ -397,6 +397,46 @@
     "old_jobs"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "729b3528-914d-4a54-bdd2-e830add5964f",
+   "metadata": {},
+   "source": [
+    "To retrieve a job's job_id by its position in the list, use `catalog.get_job_by_id()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d7e9b763-40f4-486e-8df6-e478bc7d597f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the job_id for the job at index position 1\n",
+    "jobid = old_jobs[1].job_id\n",
+    "job_by_id = catalog.get_job_by_id(jobid)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d43b0824-bcf1-4f44-8eb5-1f9e5f4ee5a9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+   "assert job_by_id.job_id == old_jobs.job_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb869150-0942-4640-a8c7-7b73c19173ba",
+   "metadata": {},
+   "source": [
+    "```\n",
+    "<Job | b1486233-3f59-4c19-b52d-6ad729b891cc>\n",
```

Comment:
@Eric-Arellano when there is no print statement, would the assert statement still give an output of any kind? (What is here is left over from when there was a print statement, so I need to update it)

Reply:
No, the `assert` does not give output. Rather than hardcoding the output as markdown like this, it's better to update the `code` block to directly call `print()` and rely on `nb-tester` (i.e. run `tox`) to generate the output.

Reply:
I've been running the notebook locally to get outputs. Should I be leaving them blank and leave it to tox to supply the outputs?

Reply:
Running it locally is fine too